root/edit/components/EditList.js

```diff
@@ -98,8 +98,14 @@ const EditList = ({
 
       {edits.length ? (
         <div id="edits">
-          <PaginatedResults guessSearch={mightBeMoreEdits} pager={pager}>
-            <form action="/edit/enter_votes" method="post">
+          <form action="/edit/enter_votes" method="post">
```

Comment:
Just as a heads-up, this might cause problems if there are plans for `PaginatedResults` to include another form within the paginator for e.g. jumping directly to a typed-in page. As far as I know, HTML5 doesn't allow nesting `form` elements (see https://html.spec.whatwg.org/multipage/forms.html#the-form-element).

Reply:
For now it doesn’t seem to be causing any problem.

Reply:
You could use the button's [form attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#form) instead to assign it to a form which is not the button's ancestor.

Reply:
True, thanks. I think that that'd require a little bit of plumbing since the attribute doesn't seem to be exposed by the `FormSubmit` component.